docs/asset-store/docs/05-01-asset-store.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Asset Store chart
+type: Configuration
+---
+
+To configure the Asset Store chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+>**TIP:** To learn more about how to use overrides in Kyma, see the following documents: 
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Top-level charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-top-level-charts-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **minio.mode** | Specifies Minio mode. For details about switching to Minio Gateway mode, see [this document](#tutorials-set-minio-to-the-google-cloud-storage-gateway-mode) | `standalone` |
```

Comment:
It is not related to Gateway, please see my comment: https://github.com/kyma-project/kyma/pull/3935#discussion_r281582702

Here is a list of MinIO chart params with description: https://github.com/helm/charts/tree/master/stable/minio#configuration

Reply:
So it is not related to Gateway in any way? Or maybe the value should be changed to `distributed` to be able to switch to the Gateway mode?